_api-reference/snapshots/get-snapshot-status.md

```diff
@@ -420,4 +428,4 @@ All property values are Integers.
 :--- | :--- | :--- |
 | shards_stats | Object | See [Shard stats](#shard-stats). |
 | stats | Object | See [Snapshot file stats](#snapshot-file-stats). |
```

Comment:
Below: After "**total**": "size" => "sizes"? Additionally, should this be a bulleted list, and should the names of the stages be formatted so as to indicate that they are names? "`started`", for example? Also please verify the capitalization of the list items.